.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/
 target/
-logs/
\ No newline at end of file
+logs/
+Hibernate-cinema.iml
```

Comment:
```suggestion
*.iml
```

Reply:
yup sorry, forgot to delete it last time